There are issues with symengine as well.  https://travis-ci.org/sympy/sympy/jobs/485558330

```
/home/travis/build/sympy/sympy/sympy/physics/mechanics/tests/test_functions.py 
Traceback (most recent call last):
  File "/home/travis/build/sympy/sympy/sympy/physics/mechanics/tests/test_functions.py", line 1, in <module>
    from sympy.core.backend import sin, cos, tan, pi, symbols, Matrix, zeros, S
  File "/home/travis/build/sympy/sympy/sympy/core/backend.py", line 6, in <module>
    from symengine import (Symbol, Integer, sympify, S,
  File "/home/travis/miniconda/envs/test-environment/lib/python3.7/site-packages/symengine/__init__.py", line 1, in <module>
    from .lib.symengine_wrapper import (Symbol, sympify as S, sympify,
ImportError: /home/travis/miniconda/envs/test-environment/lib/python3.7/site-packages/symengine/lib/symengine_wrapper.cpython-37m-x86_64-linux-gnu.so: undefined symbol: _ZN9SymEngine6expandERKNS_3RCPIKNS_5Bas
```

Hopefully they get fixed by the time tensorflow is fixed. CC @isuruf

--- a/.travis.yml
+++ b/.travis.yml
@@ -122,6 +122,8 @@ matrix:
         - TEST_SLOW="true"
         - SPLIT="2/3"
     - python: 3.7
+      dist: xenial
+      sudo: true
       env:
         - TEST_SLOW="true"
         - SPLIT="3/3"

